lab-dustin/test/fp.test.js

```diff
@@ -0,0 +1,59 @@
+const assert = require('assert');
+const { expect } = require('chai');
+const sut = require('../lib/fp.js');
+
+describe('Module', function(){
+  describe('#mapFunction()', function(){
+    it('should return new array', function(done){
+      sut.mapFunction([1,2,3], function(array){
+        expect(array).to.be.an('array');
+        var expectedArray = [2,4,6];
+        array.forEach(function(item,i){
+          assert.equal(array[i], expectedArray[i]);
```

Comment:
`array[i]` here could be `item`. `assert.deepEqual(array, expectedArray)` is also an option.